SMLHelper/Handlers/CoordinatedSpawnsHandler.cs

```diff
@@ -0,0 +1,135 @@
+        /// </summary>
+        /// <param name="techTypeToSpawn">The TechType to spawn</param>
+        /// <param name="coordinatesToSpawnTo">the coordinates the <see cref="TechType"/> should spawn to</param>
+        void ICoordinatedSpawnHandler.RegisterCoordinatedSpawnsForOneTechType(TechType techTypeToSpawn, List<Vector3> coordinatesToSpawnTo)
+        {
+            var spawnInfos = new List<SpawnInfo>();
+            foreach (var coordinate in coordinatesToSpawnTo)
+            {
+                spawnInfos.Add(new SpawnInfo(techTypeToSpawn, coordinate));
+            }
+            LargeWorldStreamerPatcher.spawnInfos.AddRange(spawnInfos);
+        }
+
+        /// <summary>
+        /// Registers a Coordinated Spawn
+        /// </summary>
+        /// <param name="spawnInfo">the SpawnInfo to spawn</param>
+        void ICoordinatedSpawnHandler.RegisterCoordinatedSpawn(SpawnInfo spawnInfo)
+        {
+            LargeWorldStreamerPatcher.spawnInfos.Add(spawnInfo);
+        }
+
+        /// <summary>
+        /// registers Many Coordinated Spawns.
+        /// </summary>
+        /// <param name="spawnInfos">The SpawnInfos to spawn.</param>
+        void ICoordinatedSpawnHandler.RegisterCoordinatedSpawns(List<SpawnInfo> spawnInfos)
+        {
+            LargeWorldStreamerPatcher.spawnInfos.AddRange(spawnInfos);
+        }
+
+        #endregion
+
+        #region Static Methods
+
+        /// <summary>
+        /// Registers a Coordinated Spawn
+        /// </summary>
+        /// <param name="spawnInfo">the SpawnInfo to spawn</param>
+        public static void RegisterCoordinatedSpawn(SpawnInfo spawnInfo)
+        {
+            Main.RegisterCoordinatedSpawn(spawnInfo);
+        }
+
+        /// <summary>
+        /// registers Many Coordinated Spawns.
+        /// </summary>
+        /// <param name="spawnInfos">The SpawnInfo to spawn.</param>
+        public static void RegisterCoordinatedSpawns(List<SpawnInfo> spawnInfos)
+        {
+            Main.RegisterCoordinatedSpawns(spawnInfos);
+        }
+
+        /// <summary>
+        /// Registers Multiple Coordinated spawns for one single passed TechType
+        /// </summary>
+        /// <param name="techTypeToSpawn">The TechType to spawn</param>
+        /// <param name="coordinatesToSpawnTo">the coordinates the <see cref="techTypeToSpawn"/> should spawn to</param>
+        public static void RegisterCoordinatedSpawnsForOneTechType(TechType techTypeToSpawn, List<Vector3> coordinatesToSpawnTo)
+        {
+            Main.RegisterCoordinatedSpawnsForOneTechType(techTypeToSpawn, coordinatesToSpawnTo);
+        }
+
+        #endregion
+    }
+    
+    #region SpawnInfo
+    /// <summary>
+    /// a basic class that provides enough info for the <see cref="CoordinatedSpawnsHandler"/> System to function.
+    /// </summary>
+    public class SpawnInfo
+    {
+        public TechType techType;
+        public string classId;
+        public Vector3 spawnPosition;
```

Comment:
Do these 3 need to be public or can they be internal?   If public then they need XML Docs

Reply:
tbh I think these should be internal as they have constructors to deal with the assignment.

Reply:
was pushing a commit for them when u sent this comment :)